go/vt/vtctl/grpcvtctldserver/server_slow_test.go

```diff
@@ -141,7 +141,7 @@ func TestEmergencyReparentShardSlow(t *testing.T) {
 					},
 					"zone1-0000000200": {
 						StopStatus: &replicationdatapb.StopReplicationStatus{
-							Before: &replicationdatapb.Status{},
+							Before: &replicationdatapb.Status{IoState: int32(mysql.ReplicationStateRunning), SqlState: int32(mysql.ReplicationStateRunning)},
```

Comment:
We're changing this — so that it's correct — all over because we now use it in the tests. Before it was unused.